packages/@okta/vuepress-site/docs/reference/api/sessions/index.md

```diff
@@ -675,49 +639,49 @@ HTTP/1.1 204 No Content
+| id                                        | A unique key for the Session                                                                    | String                                    | FALSE    | TRUE   | TRUE     |
+| login                                     | A unique identifier for the user (username)                                                     | String                                    | FALSE    | TRUE   | TRUE     |
+| userId                                    | A unique key for the [user](/docs/reference/api/users/#get-user-with-id)                                             | String                                    | FALSE    | TRUE   | TRUE     |
+| expiresAt                                 | A timestamp when the Session expires                                                                | Date                                      | FALSE    | TRUE   | TRUE     |
+| status                                    | current [status](#session-status) of the Session                                              | `ACTIVE`, `MFA_REQUIRED`, or `MFA_ENROLL` | FALSE    | TRUE   | TRUE     |
+| lastPasswordVerification                  | A timestamp when the user last performs the primary or step-up authentication with a password            | Date                                      | TRUE     | TRUE   | TRUE     |
+| lastFactorVerification                    | A timestamp when user last performs multifactor authentication                                | Date                                      | TRUE     | TRUE   | TRUE     |
+| amr                                       | Authentication method reference                                                               | [AMR object](#amr-object)                 | FALSE    | FALSE  | TRUE     |
+| idp                                       | The identity provider that is used to authenticate the user                                               | [IDP object](#idp-object)                 | FALSE    | FALSE  | TRUE     |
+| mfaActive                                 | Indicates whether the user has [enrolled an MFA factor](/docs/reference/api/factors/#list-enrolled-factors)        | Boolean                                   | FALSE    | FALSE  | TRUE     |
 
-> The `mfaActive` parameter is a <ApiLifecycle access="deprecated" /> feature. Use the `lastFactorVerification` attribute in conjunction with `amr` to understand if the user has performed MFA for the current session. Use the [Factors API](/docs/reference/api/factors/#list-enrolled-factors) to query the factor enrollment status for a given user.
+> The `mfaActive` parameter is a <ApiLifecycle access="deprecated" /> feature. Use the `lastFactorVerification` attribute in conjunction with `amr` to understand if the user has performed MFA for the current Session. Use the [Factors API](/docs/reference/api/factors/#list-enrolled-factors) to query the factor enrollment status for a given user.
```

Comment:
regarding these items with > at the beginning. If you look in the live docs, you see they have the box around them like a note, but without the actual WORD Note. I've been converting these to Note format. If you see them, can you please convert them, also? Thanks!